authors/apps/articles/urls.py

```diff
@@ -9,4 +9,6 @@
          name='articles'),
     path('<slug>', views.ArticleDetailApiView.as_view(),
          name='article-details'),
+    path('<slug>/favorite', views.ToggleFavoriteAPIView.as_view(),
```

Comment:
@oma0256 we have been following the naming convention on the urls with the trailing slash. Could you please revise this

Reply:
@missvicki we're actually currently not using the trailing slash on our urls

Reply:
True @oma0256 .